### ISSUE TITLE
navigate between articles

### ISSUE THREAD

**Issue by nicosomb** [wallabag/android-app#122]
When I'm reading an article, I want to swipe to read the next article.

**Comment by tcitworld** [comment]
This may be tricky since content is displayed in a webview.

**Comment by nicosomb** [comment]
Why do we use webview?

**Comment by tcitworld** [comment]
Hmm, it's quite the best option we have to display stylised text. Allows also YouTube videos to be read.

**Comment by nicosomb** [comment]
Can you have a look to the Reflets.info app ? Is that possible to implement this way of view and swipe ?

**Comment by di72nn** [comment]
[The app](https://github.com/freebyte/Reflets-android/blob/master/src/info/reflets/app/ArticleActivity.java) uses a couple of predefined views (title, date, author, etc.) and generates main content view via [Html.fromHtml](https://developer.android.com/reference/android/text/Html.html#fromHtml%28java.lang.String,%20android.text.Html.ImageGetter,%20android.text.Html.TagHandler%29). I believe WebView is more suitable currently.

If I understand correctly, "swipe" is already implemented (*more or less*).

**Comment by nicosomb** [comment]
Yes, swipe is already implemented, but it's not very convenient to use.

**Comment by di72nn** [comment]
Sorry to duplicate (issues)[#134], but this one leads me into wrong direction :)
